crates/bevy_pbr/src/prepass/mod.rs

```diff
@@ -512,7 +592,7 @@ where
                 conservative: false,
             },
             depth_stencil: Some(DepthStencilState {
-                format: DEPTH_PREPASS_FORMAT,
+                format: TextureFormat::Depth32Float,
```

Comment:
Core 3D depth format? Or meant to be hard-coded? If intended, maybe a comment explaining why?

Reply:
Good catch. I think this got lost in the shuffle when I moved away from using the stencil buffer.